src/history/eventraisedevent.ts

```diff
@@ -15,7 +15,12 @@ export class EventRaisedEvent extends HistoryEvent {
             options.timestamp,
         );
 
-        this.Name = options.name;
-        this.Input = options.input;
+        if (options.name === undefined) {
+            throw new Error("EventRaisedEvent needs a name provided.");
+        } else {
+            this.Name = options.name;
+        }
+
+        this.Input = options.input !== null ? options.input : undefined;
```

Comment:
Just so I understand, this basically coerces any `null` values into `undefined`?

Reply:
That is exactly what this is doing.